concept/type/AttributeTypeImpl.java

```diff
@@ -49,36 +49,31 @@
 
     private static final Label ROOT_LABEL = Label.of("attribute");
 
-    AttributeTypeImpl(java.lang.String label, boolean isRoot) {
```

Comment:
Label was not consistently used across our Concept constructors, which made code harder to reuse.